etc/adminhtml/system/adyen_advanced_order_processing.xml

```diff
@@ -40,7 +40,7 @@
         </field>
         <field id="auto_capture_openinvoice" translate="label" type="select" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
             <label>Use auto-capture for OpenInvoice payments</label>
-            <tooltip>Applicable for Klarna, Afterpay Touch, Ratepay, FacilyPay/Oney, Affirm, Clearpay, Zip and PayBright. By default, Open Invoice methods are set to manual capture. If you want auto capture you need to contact magento@adyen.com. After approval has been given you can set this option to 'Yes'.</tooltip>
+            <tooltip>Applicable for Klarna, Afterpay, Afterpay Touch, Ratepay, FacilyPay/Oney, Affirm, Clearpay, Zip and PayBright. By default, Open Invoice methods are set to manual capture. If you want auto capture you need to contact magento@adyen.com. After approval has been given you can set this option to 'Yes'.</tooltip>
```

Comment:
Maybe change the email address like in #1538

Reply:
Thank you @tnaber , this is the [commit](https://github.com/Adyen/adyen-magento2/pull/1555/commits/4b36df44e14e807c428eda8f23342555c76c10be)